src/initialization.js

```diff
@@ -69,21 +69,31 @@ var initialization = {
 
         common.consentPayloadDefaults =
             common.consentHandler.getConsentSettings();
-        var initialConsentState = common.consentHandler.getUserConsentState();
 
-        var defaultConsentPayload =
-            common.consentHandler.generateConsentStatePayloadFromMappings(
-                initialConsentState,
-                common.consentMappings
-            );
+        var defaultConsentPayload = common.cloneObject(common.consentPayloadDefaults)
+        var updatedConsentState = common.consentHandler.getUserConsentState();
+        var updatedDefaultConsentPayload =
+        common.consentHandler.generateConsentStatePayloadFromMappings(
+            updatedConsentState,
+            common.consentMappings
+        );
```

Comment:
also this can become one `var` and then comma separated instead of having 3 vars for each, let me know if this is preferred

Reply:
moved it back to 3 vars instead of comma separated per meeting with @alexs-mparticle